events/eventservices/directoryBlockMapper.go

```diff
@@ -17,7 +20,6 @@ func mapDirectoryBlockHeader(header interfaces.IDirectoryBlockHeader) *eventmess
 	result := &eventmessages.DirectoryBlockHeader{
 		BodyMerkleRoot:        header.GetBodyMR().Bytes(),
 		PreviousKeyMerkleRoot: header.GetPrevKeyMR().Bytes(),
-		PreviousFullHash:      header.GetPrevFullHash().Bytes(),
```

Comment:
Why is previous full hash removed: It exists in the doc https://github.com/FactomProject/FactomDocs/blob/master/factomDataStructureDetails.md#directory-block

Reply:
The doc describes what happens inside the marshallBinary() function en FullHash is the hash over the result of the marshallBinary(). We let protobuf do the marshalling so there is no point sending this field.